packages/dev/package.json

```diff
@@ -14,7 +14,7 @@
     "tsc": "tsc",
     "truffle-compile": "truffle compile",
     "lint": "eslint -f unix .",
-    "test": "lerna run tsc && truffle --compile-all --network npmtest test",
+    "test111": "lerna run tsc && truffle --compile-all --network npmtest test",
```

Comment:
Are you sure this is a name you want?

Reply:
This is a temporaray name, Stop reviewing this PR :rofl: